tests/test_web/test_api_v1.py

```diff
@@ -317,10 +308,6 @@ def test_add_bundles_overwrite_not_allowed(mock_smfsc, client, db):
 @pytest.mark.parametrize(
     'data, error_msg',
     (
-        (
-            {'from_index': 'pull:spec', 'binary_image': 'binary:image', 'add_arches': ['s390x']},
-            '"operators" should be a non-empty array of strings',
```

Comment:
Accidental removal?

Reply:
no, because that is a valid case now and should not fail right?

Reply:
This is for the `test_rm_operators_invalid_params_format` test and the error message is about the `operators` parameter. How is that related to these changes?